pkg/apiserver/webhooks/validating-webhook/validating-webhook.go

```diff
@@ -206,6 +207,24 @@ func validateDataVolumeSpec(field *k8sfield.Path, spec *cdicorev1alpha1.DataVolu
 		return causes
 	}
 
+	accessModes := spec.PVC.AccessModes
+	if len(accessModes) > 1 {
```

Comment:
Note kubernetes itself will return an error if there are no access modes, so no need to check for it here.